tests/integration/component.go

```diff
@@ -0,0 +1,297 @@
+package integration
```

Comment:
each integration test should be in a separate package.
`package component`

Reply:
Done